ui/component/common/form-components/form-field.jsx

```diff
@@ -236,48 +253,51 @@ export class FormField extends React.PureComponent<Props> {
         case 'textarea':
           return (
             <fieldset-section>
-              {(label || quickAction) && (
-                <div className="form-field__two-column">
-                  <label htmlFor={name}>{label}</label>
-                  {quickAction}
```

Comment:
Doesn't this break the functionality of the "quick action"?  I understand it is not used today since we have disabled the Advanced/Simple editor toggle, but the functionality shouldn't be broken. e.g. if I want to add a button there like "show instructions" for a specific text field, "quick action" is intended for that but will not work ...
Also, the ` || quickAction` surrounding it no longer makes sense.

Killing the feature is an option if it's no longer useful, but remove it completely if so (can be done later since I understand we have lack of time).

Reply:
Yeah I thought it could break the `two-column` thing by adding another item there, but for now it works on both sides so the css will have to be changed if `quickAction` is used. this was reverted